prediction-applier/src/integration-test/java/org/alfresco/hxi_connector/prediction_applier/util/DockerTags.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.hxi_connector.prediction_applier.util;
+
+import java.io.InputStream;
+import java.nio.file.NoSuchFileException;
+import java.util.NoSuchElementException;
+import java.util.Properties;
+import java.util.Set;
+
+import lombok.AccessLevel;
+import lombok.Cleanup;
+import lombok.NoArgsConstructor;
+import lombok.SneakyThrows;
+
+/**
+ * Allows to access properties from 'docker-tags.properties' file. It's required to run: `mvn package` before accessing the properties file, to allow Maven filter out and replace variables.
+ */
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class DockerTags
```

Comment:
This will be the third class named `DockerTags` in the project. From what I can see, there are at least several methods which seem to be either exactly the same or have same purpose.
Have we thought about creating a common-codebase module inside the project? That could also be of use for node filtering.

Reply:
You're probably right. We did think about it before, but there wasn't much duplication. Now I can think of several things:
* `DockerTags` and other Testcontainers code
* Constant for names of properties like `cm:content`
* Prediction model
* Bulk ingestion -> Live ingestion message format? (I don't remember seeing duplication of code here, but I think there must be)

Reply:
I was thinking the same while copying this class, but lets do it in separate PR